### ISSUE TITLE
Transport properties of water

### ISSUE THREAD

**Issue by jowr** [CoolProp/CoolProp#390]
It looks like thermal conductivity and viscosity only depend on pressure, is that correct? The values differ quite a bit from the incompressible data.
Figures 2 and 4 in http://scitation.aip.org/content/aip/journal/jpcrd/41/3/10.1063/1.4738955 also suggest that there might be an error. 
 Any comments anyone?

![h2ocomp_l](https://cloud.githubusercontent.com/assets/769593/5617596/be8272de-9510-11e4-8032-0735f6f8103b.jpg)
![h2ocomp_v](https://cloud.githubusercontent.com/assets/769593/5617602/c6abc500-9510-11e4-98d9-454aa22640ab.jpg)
![nbs_fitreport](https://cloud.githubusercontent.com/assets/769593/5617632/f8d50fdc-9510-11e4-9fe9-2f57ac643ac7.jpg)

**Comment by ibell** [comment]
Yes that is clearly an error.

**Comment by ibell** [comment]
Weird. For me in python:

```python
In [10]: CoolProp.CoolProp.PropsSI('L','T',500,'P',70e6,'Water')
Out[10]: 0.6974296954778116

In [11]: CoolProp.CoolProp.PropsSI('L','T',700,'P',70e6,'Water')
Out[11]: 0.44961331422897943

In [12]: CoolProp.__version__
Out[12]: u'5.0.5'

In [13]: CoolProp.__gitrevision__
Out[13]: u'984a8f98d79ca890d4d47540670b6918c9d41ed5'
```

**Comment by ibell** [comment]
Is it possible that there is a problem with your plotting code? Can you provide the code for the plots with the horizontal lines?

**Comment by jowr** [comment]
I cannot reproduce your results. It looks like there is a problem with the state class. The first value is always correct, but the transport properties do not get updated in the subsequent calls.
```
from __future__ import print_function, division
import numpy as np
import CoolProp
from CoolProp.CoolProp import PropsSI

print(CoolProp.__version__)
print(CoolProp.__gitrevision__)

points = 4
fluid='water'
Tmin =   0 + 273.15 # PropsSI('Tmin',fluid)+1e-3
Tmax = 200 + 273.15 # PropsSI('Tmax',fluid)-1e-3
Pmin =   1e5 # PropsSI('pmin',fluid)+1e-0
Pmax = 100e5 # PropsSI('pmax',fluid)-1e-0

T = np.linspace(Tmin, Tmax, points)
p = np.linspace(Pmin, Pmax, points)
for l in ["V","L"]:
    print("{0}: {1}".format("T",T))
    for pi in p:
        print("{0}: {1} at {2} bar".format(l,PropsSI(l,"T",T,"P",pi,fluid),pi/1e5))

Tmin = 500
Tmax = 700
Pmin = 500e5
Pmax = 700e5

T = np.linspace(Tmin, Tmax, points)
p = np.linspace(Pmin, Pmax, points)
for l in ["V","L"]:
    print("{0}: {1}".format("T",T))
    for pi in p:
        print("{0}: {1} at {2} bar".format(l,PropsSI(l,"T",T,"P",pi,fluid),pi/1e5))
```
```
5.0.6dev
83a670a4fa142416e13444f54c95e162bb55d26b
T: [ 273.15        339.81666667  406.48333333  473.15      ]
V: [ 0.00179176  0.00179176  0.00179176  0.00179176] at 1.0 bar
V: [ 0.00178447  0.00178447  0.00178447  0.00178447] at 34.0 bar
V: [ 0.00177742  0.00177742  0.00177742  0.00177742] at 67.0 bar
V: [ 0.00177059  0.00177059  0.00177059  0.00177059] at 100.0 bar
T: [ 273.15        339.81666667  406.48333333  473.15      ]
L: [ 0.55564865  0.55564865  0.55564865  0.55564865] at 1.0 bar
L: [ 0.55814409  0.55814409  0.55814409  0.55814409] at 34.0 bar
L: [ 0.56060201  0.56060201  0.56060201  0.56060201] at 67.0 bar
L: [ 0.56302293  0.56302293  0.56302293  0.56302293] at 100.0 bar
T: [ 500.          566.66666667  633.33333333  700.        ]
V: [ 0.00012926  0.00012926  0.00012926  0.00012926] at 500.0 bar
V: [ 0.00013071  0.00013071  0.00013071  0.00013071] at 566.666666667 bar
V: [ 0.00013213  0.00013213  0.00013213  0.00013213] at 633.333333333 bar
V: [ 0.00013353  0.00013353  0.00013353  0.00013353] at 700.0 bar
T: [ 500.          566.66666667  633.33333333  700.        ]
L: [ 0.68134801  0.68134801  0.68134801  0.68134801] at 500.0 bar
L: [ 0.68679445  0.68679445  0.68679445  0.68679445] at 566.666666667 bar
L: [ 0.6921533  0.6921533  0.6921533  0.6921533] at 633.333333333 bar
L: [ 0.6974297  0.6974297  0.6974297  0.6974297] at 700.0 bar
```

**Comment by jowr** [comment]
I think this is a Python problem, see #394

**Comment by romarro** [comment]
there's still a problem, and it has to do with the incompresible water, the NBS. Below you can see my code:
In [41]: PropsSI('V','T',85.+273.15,'P',2e5,'Water')
Out[41]: 0.0003331021692555941

In [42]: PropsSI('V','T',85.+273.15,'P',2e5,'INCOMP::NBS')
Out[42]: 0.03339785017964899

In [43]: import CoolProp

In [44]: CoolProp.__version__
Out[44]: u'5.1.1'

In [45]: CoolProp.__gitrevision__
Out[45]: u'12f006445f234e572e64cc820146ab5d2c2a9d10'

And a request. It is possible to implement in PropsSI also the Prandtl number?

**Comment by ibell** [comment]
Confirm still a problem in dev:
```

>>> from CoolProp.CoolProp import PropsSI

>>> PropsSI('V','T',85.+273.15,'P',2e5,'INCOMP::NBS')

0.03339785017964899

>>> PropsSI('V','T',85.+273.15,'P',2e5,'Water')

0.0003331021692555946

>>> import CoolProp

>>> CoolProp.__version__

u'5.1.2dev'

>>> CoolProp.__gitrevision__

u'129381efbb5a5cae7dc79019347ab4fe2ad8b109'

```

**Comment by ibell** [comment]
Prandtl is included in development version of 5.1.2:
http://www.coolprop.org/dev/coolprop/HighLevelAPI.html#table-of-string-inputs-to-propssi-function
